### PR TITLE
Redesign profile page

### PR DESCRIPTION
**Pull request by mateuuszzzzz**
PR redesigns profile page and adds some new user associated data.
**IMPORTANT: Should be merged after https://github.com/skyhitz/backend/pull/85**

- Background photo
- Instagram 
- Twitter

Beatmaker desktop/mobile:
Screenshot 2022-12-14 at 18 48 08
Screenshot 2022-12-14 at 18 50 27

Profile dekstop/mobile:
Screenshot 2022-12-15 at 15 35 45

Screenshot 2022-12-15 at 15 36 29

Edit desktop/mobile
edit_desktop

edit_mobile

### CODE REVIEW

**Review by alejomendoza — APPROVED** [review]

File: packages/app/config/index.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const testingStagingModeOn =
 
 const prodModeOn = process.env.NEXT_PUBLIC_EXPO_SKYHITZ_ENV === "production";
 
-let config: ConfigInterface = StagingConfig;
+let config: ConfigInterface = DevelopmentConfig;
```
Comment (jmysliv) [inline packages/app/config/index.ts:13]:
```suggestion
let config: ConfigInterface = StagingConfig;
```

File: packages/app/features/dashboard/beatmaker/index.tsx [hunk 2]
```diff
@@ -36,28 +37,44 @@ export default function BeatmakerScreen() {
...
+      {user!.data?.description && (
+        <>
+          <Text className="w-full max-w-6xl mx-auto pl-4 mt-6 text-lg font-bold">
+            Bio
+          </Text>
+          <ScrollView className="h-1/10">
+            <Text className="w-full max-w-6xl mx-auto pl-4 mt-3 text-lg">
+              {user!.data?.description}
+            </Text>
+          </ScrollView>
```
Comment (jmysliv) [inline packages/app/features/dashboard/beatmaker/index.tsx:74]: why do we need scrollview here?

File: packages/app/features/dashboard/profile/edit/ChangeImages.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    }
+  }, [error, toast]);
+
+  return (
+    <Button
+      disabled={!permissionGranted || loading}
+      loading={loading}
+      onPress={pickMedia}
+      text={text}
+      // TODO change size to small?
```
Comment (jmysliv) [inline packages/app/features/dashboard/profile/edit/ChangeImages.tsx:74]: can you either try it and check if it looks better or remove it?

File: packages/app/features/dashboard/profile/edit/index.tsx [hunk 4]
```diff
@@ -117,6 +131,20 @@ export default function EditProfileScreen() {
...
             />
+            <Line />
+            <FormInputWithIcon
+              value={values.twitter ?? ""}
+              placeholder="Twitter username"
+              onChangeText={handleChange("twitter")}
+              icon={Twitter}
+            />
+            <Line />
+            <FormInputWithIcon
```
Comment (jmysliv) [inline packages/app/features/dashboard/profile/edit/index.tsx:142]: `FormInputWithIcon` has an error prop. Can we use it and remove displaying errors at the bottom?

File: packages/app/ui/icons/image.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Svg, { Path, Rect, Circle } from "react-native-svg";
+
+function Image({ size = 24, color = "#000000" }) {
```
Comment (jmysliv) [inline packages/app/ui/icons/image.tsx:4]: Do we need it?

File: packages/app/validation/index.tsx [hunk 6]
```diff
@@ -122,6 +124,15 @@ export const validateProfilePicture = (image: MediaFileInfo): string | null => {
   );
 };
 
+export const validateBackgroundImage = (
+  image: MediaFileInfo
+): string | null => {
+  return (
+    //TO DO: Add validation rules for image size/ratio etc.
```
Comment (jmysliv) [inline packages/app/validation/index.tsx:131]: unresolved todo

File: packages/app/features/dashboard/beatmaker/index.tsx (hunk 2 above)
Comment (jmysliv) [reply packages/app/features/dashboard/beatmaker/index.tsx:74]: I would go with the characters limit for bio.

File: packages/app/ui/icons/image.tsx (hunk 5 above)
Comment (jmysliv) [reply packages/app/ui/icons/image.tsx:4]: okay, could you remove it?

**Review by jmysliv — APPROVED** [review]

**Pull request merged**